packages/utils/test/normalize.test.ts

```diff
@@ -366,28 +366,26 @@ describe('normalize()', () => {
     });
 
     test('primitive values in objects/arrays', () => {
-      expect(normalize(['foo', 42, undefined, NaN])).toEqual(['foo', 42, '[undefined]', '[NaN]']);
+      expect(normalize(['foo', 42, NaN])).toEqual(['foo', 42, '[NaN]']);
```

Comment:
I'm just thinking why not keep the undefined here?

```suggestion
      expect(normalize(['foo', 42, undefined, NaN])).toEqual(['foo', 42, undefined, '[NaN]']);
```

I would expect this to pass, or not?

Reply:
youre right!